lib/facade_converter.ts

```diff
@@ -88,6 +88,36 @@ export class FacadeConverter extends base.TranspilerBase {
     return handler && !handler(pa);
   }
 
+  buildImports(sourceFile: ts.SourceFile) {
+    var imports: {[type: string]: boolean} = {};
+
+    function getNodes(sourceFile: ts.SourceFile): ts.Node[] {
+      var nodes: ts.Node[] = [];
```

Comment:
This will be rather expensive for a large file, would it be nicer to pass in a predicate directly? Or maybe just have a more specific `findTypeNames` instead of this very generic function?